core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/centos-7/requirements.txt

```diff
@@ -149,6 +148,8 @@ kubernetes-cni-0.7.5-0
 kubernetes-cni-0.8.6-0
 
 [files]
+# Github repository for erlang rpm is used since packagecloud repository is limited to a certain number of packages and erlang package from erlang-solution repository is much more complex and bigger
```

Comment:
```suggestion
# Github repository for erlang rpm is used since packagecloud repository is limited to a certain number of versions and erlang package from erlang-solutions repository is much more complex and bigger
```